src/scenic/syntax/veneer.py

```diff
@@ -111,1000 +132,1341 @@
+    return normalizeAngle(X.yaw - Y.yaw) 
 
 def ApparentHeading(X, Y=None):
-	"""The :grammar:`apparent heading of <oriented point> [from <vector>]` operator.
+    """The :grammar:`apparent heading of <oriented point> [from <vector>]` operator.
 
-	If the :grammar:`from <vector>` is omitted, the position of ego is used.
-	"""
-	if not isinstance(X, OrientedPoint):
-		raise RuntimeParseError('"apparent heading of X from Y" with X not an OrientedPoint')
-	if Y is None:
-		Y = ego()
-	Y = toVector(Y, '"relative heading of X from Y" with Y not a vector')
-	return apparentHeadingAtPoint(X.position, X.heading, Y)
+    If the :grammar:`from <vector>` is omitted, the position of ego is used.
+    """
+    if not isA(X, OrientedPoint):
+        raise TypeError('"apparent heading of X from Y" with X not an OrientedPoint')
+    if Y is None:
+        Y = ego()
+    Y = toVector(Y, '"relative heading of X from Y" with Y not a vector')
+    return apparentHeadingAtPoint(X.position, X.heading, Y)
 
 def DistanceFrom(X, Y=None):
-	"""The :scenic:`distance from {X} to {Y}` polymorphic operator.
+    """The :scenic:`distance from {X} to {Y}` polymorphic operator.
 
-	Allowed forms::
+    Allowed forms::
 
-		distance from <vector> [to <vector>]
-		distance from <region> [to <vector>]
-		distance from <vector> to <region>
+        distance from <vector> [to <vector>]
+        distance from <region> [to <vector>]
+        distance from <vector> to <region>
 
-	If the :grammar:`to <vector>` is omitted, the position of ego is used.
-	"""
-	X = toTypes(X, (Vector, Region), '"distance from X to Y" with X neither a vector nor region')
-	if Y is None:
-		Y = ego()
-	Y = toTypes(Y, (Vector, Region), '"distance from X to Y" with Y neither a vector nor region')
-	return X.distanceTo(Y)
+    If the :grammar:`to <vector>` is omitted, the position of ego is used.
+    """
+    X = toTypes(X, (Vector, Region), '"distance from X to Y" with X neither a vector nor region')
+    if Y is None:
+        Y = ego()
+    Y = toTypes(Y, (Vector, Region), '"distance from X to Y" with Y neither a vector nor region')
+    return X.distanceTo(Y)
 
 def DistancePast(X, Y=None):
-	"""The :grammar:`distance past <vector> of <oriented point>` operator.
+    """The :grammar:`distance past <vector> of <oriented point>` operator.
 
-	If the :grammar:`of {oriented point}` is omitted, the ego object is used.
-	"""
-	X = toVector(X, '"distance past X" with X not a vector')
-	if Y is None:
-		Y = ego()
-	Y = toType(Y, OrientedPoint, '"distance past X of Y" with Y not an OrientedPoint')
-	return Y.distancePast(X)
+    If the :grammar:`of {oriented point}` is omitted, the ego object is used.
+    """
+    X = toVector(X, '"distance past X" with X not a vector')
+    if Y is None:
+        Y = ego()
+    Y = toType(Y, OrientedPoint, '"distance past X of Y" with Y not an OrientedPoint')
+    return Y.distancePast(X)
 
+# TODO(shun): Migrate to `AngleFrom`
```

Comment:
@shumbo What's this comment about?

Reply:
@dfremont My understanding is that `angle to X` is the same as `angle from ego to X`. Inserting `from ego` can now be done in the compiler, so I thought we can unify `AngleTo` and `AngleFrom` into a single function. Am I correct here?

Reply:
OK, we can leave this in for now then.